packages/fds-components/src/style/index.css

```diff
@@ -5,6 +5,7 @@
 @import "fds-icons/lib/fds-icons";
 
 /* Components */
+@import "./buttons/buttonGroup.css";
```

Comment:
Do you think this should go in `interactive/buttonGroup`, or buttons should be their own category?

If we want to make a new category for buttons, I can file a ticket to reclassify the stuff in "interactive".

Reply:
I actually didn't notice the interactive. I think that makes sense to go there 👍